### PR TITLE
SL-1550: v2 docs

### PR DESCRIPTION
**Pull request by mkrvaric**
# Links (task, documents, etc.)
- https://zypone.atlassian.net/browse/SL-1550
# Problem
- 
# Solution
- 
# Additional info
-

### CODE REVIEW

File: source/includes/_actions.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
+# Actions
+
+## Get All Actions
+
+```shell
+curl --request GET \
+  --url 'https://public-api.lumiformapp.com/api/v2/actions' \
```
Comment (seyfer) [inline source/includes/_actions.md:7]: wasn't this task to keep v1 docs as well? I think we need to have v1 and v2 sections, both of them

Comment (mkrvaric) [reply source/includes/_actions.md:7]: @seyfer Yes, v1 routes are still here, marked as deprecated. Issues are renamed to /v2/actions

Comment (seyfer) [reply source/includes/_actions.md:7]: @mkrvaric I am talking about docs, not routes. docs for v1 and for v2 should be accessible

Comment (mkrvaric) [reply source/includes/_actions.md:7]: @seyfer Yes, this is still here, for example: https://github.com/lumiform/slate/blob/SL-1550-update-urls/source/includes/_issues.md

Comment (seyfer) [reply source/includes/_actions.md:7]: @mkrvaric ok got it 👍

**Review by seyfer — APPROVED** [review]

**Review by CharlieBrownCharacter — APPROVED** [review]

File: source/includes/_actions.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+  --header 'Content-Type: application/json' 
+```
+
+> The above command returns JSON structured like this:
+
+```json
+{
+  "data": {
+    "id": 1,
+    "title": "Check the droid attack on the Wookies",
```
Comment (seyfer) [inline source/includes/_actions.md:163]: I really love star wars topic in docs :D

File: source/includes/_forms.md [hunk 3]
```diff
@@ -0,0 +1,372 @@
...
+  --header 'Content-Type: application/json' 
+```
+
+> The above command returns JSON structured like this:
+
+```json
+{
+  "data": {
+    "id": 1,
+    "title": "Safety checklist",
```
Comment (seyfer) [inline source/includes/_forms.md:196]: Not star wars topic enough

**Review by seyfer — APPROVED** [review]
👍  let the Force be with you

File: README.md [hunk 4]
```diff
@@ -6,6 +6,12 @@
 
 <p align="center"><img src="./source/images/example.png" width=700 alt="Screenshot of Example Documentation created with Slate"></p>
 
+Setting up local dev with docker
```
Comment (tributeee) [inline README.md:9]: Nice! This was def. missing.

File: source/includes/_actions.md [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+      "created_at": 1598649200,
+      "assignees": [
+        {
+          "id": 2,
+          "name": "Yoda",
+          "email": "yoda@jediknights.com"
+        }
+      ],
+      "status": "in_progress",
+      "due_at": 1630185200,
```
Comment (tributeee) [inline source/includes/_actions.md:37]: Just a note: If we return data in snake case always, please make sure to also convert keys received from API in later usage from camel to snake. We need to keep this API consistent as it's public, used by our clients, and we need to be professional about it 🙂 
I know you know, but doesn't hurt to mention. 😉

**Review by tributeee — APPROVED** [review]

**Pull request merged**